style/battle.css

```diff
@@ -789,6 +789,13 @@ License: GPLv2
 	background: #333;
 	color: #DDD;
 }
+.boosttext {
```

Comment:
Okay, time to nitpick class names.

I think `stat-boosted` and `stat-lowered` might be best here.